### PR TITLE
Add get highest value project

### PR DESCRIPTION
**Pull request by jericoatos**

### CODE REVIEW

**Review by TaylorB-softdevkainos — APPROVED** [review]

File: src/main/java/org/example/controllers/ProjectController.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.sql.SQLException;
+
+@Api("Project API")
+@Path("/api")
```
Comment (mlynch147) [inline src/main/java/org/example/controllers/ProjectController.java]: can we have the class level path set to /api/projects

File: src/main/java/org/example/controllers/ProjectController.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    ProjectService projectService;
+
+    public ProjectController(final ProjectService projectService) {
+        this.projectService = projectService;
+    }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/highestvalueproject")
```
Comment (mlynch147) [inline src/main/java/org/example/controllers/ProjectController.java]: change path to /highestValue

File: src/main/java/org/example/daos/ProjectDao.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ProjectDao {
+
+    public List<Project> getHighestValueProject() throws SQLException {
```
Comment (mlynch147) [inline src/main/java/org/example/daos/ProjectDao.java]: This should only return a single Project, not a List...

File: src/main/java/org/example/services/ProjectService.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+import java.sql.SQLException;
+import java.util.List;
+
+public class ProjectService {
+    ProjectDao projectDao;
+    public ProjectService(final ProjectDao projectDao) {
+        this.projectDao = projectDao;
+    }
+
+    public List<Project> getHighestValueProject() throws SQLException {
```
Comment (mlynch147) [inline src/main/java/org/example/services/ProjectService.java]: This only needs to return a single Project, not a list of projects

**Review by mlynch147 — CHANGES_REQUESTED** [review]

**Review by mlynch147 — APPROVED** [review]

**Pull request merged**